modules/cloud-getting-support.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * service_description/rhacs-cloud-service-service-description.adoc
+:_content-type: CONCEPT
+[id="cloud-getting-support_{context}"]
+= Getting support for RHACS Cloud Service
+
+The {product-title-managed-short} includes Red Hat Standard and Premium support, which you can access using the link:https://access.redhat.com/support[Red Hat Customer Portal].
+You can open support tickets for the product "Red Hat Advanced Cluster Security Cloud Service".
+
+Red Hat support responds to support tickets submitted by Limited Availability customers, while Red Hat Site Reliability Engineers (SREs) proactively monitor the health of {rh-rhacscs-first}.
```

Comment:
Doron, please let me know what changes you want.

Reply:
Actually we decided to open a separate PR for the "limited availability/support" issue: https://github.com/openshift/openshift-docs/pull/78157